applications/dappworld_learning_ecosystem.md

```diff
@@ -0,0 +1,163 @@
+# Project Name - DApp World
```

Comment:
This file is already added, if there is new content, please modify it through pull request.